README.md

```diff
@@ -53,73 +54,126 @@ If you want to create a repository with the same structure as this template you
+There's no "default" for `config_user.yaml` because it depends on system specifications and user preferences. Three things we do recommend keeping in `config_user.yaml` are the name of your Stata executable, the location of a [SCons cache directory](http://scons.org/doc/2.0.1/HTML/scons-user/c4213.html), and the location of a release directory. These fields don't have to be specified if you're not using them, and we'll prompt you for their values at runtime if you've forgotten to specify them and they're necessary. A Mac example where Example_User is running a factory-fresh StataMP and has local access to directories named cache/template and release on Dropbox would be 
 
 ```YAML
 stata_executable: stata-mp
 cache_directory: /Users/Example_User/Dropbox/cache/template
 release_directory: /Users/Example_User/Dropbox/release/
 ```
 
-##### How do I handle data external to my repository?
+#### What is `config_global.yaml`?
+
+The `config_global.yaml` tracks paths, specifications, variables, and software checks that are constant across users. We treat this file in the same manner as `config_user.yaml`, except that we do version `config_global.yaml`.
+
+#### How do I handle data external to my repository?
 
 We are agnostic about how you incorporate external data into the template. There's no custom builder for these assets, by design. Our suggestions:
 
 * When a large dataset is stored locally, `config_user.yaml` can include an entry specifying the user-specific path to that dataset. The key of the entry should be constant across users and documented in the top-level readme of the repository.
 
 * When a large dataset is stored externally, there are a few options. 
     * The top-level readme can specify manual download and storage instructions. This is simple, easy to customize, and unlikely to cause errors during a SCons build. It does, however, require each user to successfully download the same dataset, perhaps in an unstructured manner. 
-    * The download can be incorporated into the SCons build. We either execute a program to transfer data (e.g., `rsync` or `rclone`) directly in a standard SCons command or from within a script executed by one of our custom builders. These methods have the benefits of automation and dependency tracking, but they can introduce idiosyncratic errors if the download steps are prone to failure.
+    * The download can be incorporated into the SCons build. We either execute a program to transfer data (e.g., `rsync` or `rclone`) using our "anything build" or from within a script executed by one of our other custom builders. These methods have the benefits of automation and dependency tracking, but they can introduce idiosyncratic errors if the download steps are prone to failure.
```

Comment:
`_build_anything` function instead of "anything build"?

Reply:
Meant to say "anything builder" to mirror e.g., Python builder. You cool with that?

Reply:
let's capitalized/distinguish "anything", or anything-builder.

Reply:
Capitalized A in anything.